PWGJE/macros/AddTaskESDFilterPWGJETrain.C

```diff
@@ -1,4 +1,10 @@
+#ifdef __CLING__
+// ROOT6, bindet andere macros wie Header files ein, include Pfad kann mit R__ADD_INCLUDE_PATH hinzugefügt werden
+R__ADD_INCLUDE_PATH($ALICE_PHYSICS)
+#include <PWGJE/macros/CreateTrackCutsPWGJE.C>
 
+#include "AliAnalysisTaskESDfilter.h"
```

Comment:
Please do not include headers of classes in ROOT/AliRoot/AliPhysics. Those are handled automatically by cling via so-called pre-compiled headers. Includes should be used only in 2 cases:
- Code using functions from other macros (this is the case for CreateTrackCutsPWGJE.C)
- Code using functions from libraries for which no pre-compiled headers exists (examples: direct used of fastjet, pythia, other c++ generators in macros)
`AliAnalysisTaskESDfilter` is in libANALYSISalice, for which pre-compiled headers exist, ROOT should find them without including the header file.

Reply:
Done